modules/Quiz/resources/IVQScreenTemplate.js

```diff
@@ -63,11 +63,16 @@
                 $(".icon-close").css("display", "none");
                 $(_this.emptyScreen()).hide().append(
                     '<div class="ivqContainer">'+
+                    '   <div class="feedback-modal">' +
+                    '       <span class="icon-close" aria-label="Close feedback" tabindex="0" role="button"></span>' +
```

Comment:
Locale - try to see if we can pass this via the translation function 
Also applies for the "feedback" button